src/content/docs/browser/browser-integrations/mexn-integration.mdx

```diff
@@ -0,0 +1,84 @@
+</figcaption>
+
+## Install [#install]
+
+### Step 1 (optional): Install the infrastructure agent [#infra-install]
+
+Installing the infrastructure agent is optional, but doing this will get you important metrics about the host where your app is running. 
+
+To do this, follow [the infrastructure agent install steps](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent).
+
+### Step 2: Install MongoDB quickstart
+
+1. Check out our MongoDB agent requirements in our [documentation](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/mongodb-monitoring-integration/)  before installing the agent.
+2. Open the [MongoDB quickstart installation](https://newrelic.com/instant-observability/mongodb). 
+3. Click  **install now** at the bottom of the page to start the MongoDB agent installation.
+
+### Step 3: Install Node.js agent
+
+1. Check out our Node.js agent requirements in our [documentation](/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent/).
+2. Open the [Node.js quickstart installation](https://newrelic.com/instant-observability/node-js). 
+3. Click  **install now** at the bottom of the page to start the Node.js agent installation.
+
+### Step 4: Install the browser agent [#browser-install] 
+
+To install our browser agent: 
+
+1. From [one.newrelic.com](https://one.newrelic.com), go to the [**Add data** page](https://one.newrelic.com/marketplace). 
+2. Click **Browser and mobile**, then click **Browser monitoring**.
+3. Select the **Select copy/paste JavaScript code** option (for more about this code, see [our browser install docs](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent)). 
+4. Complete the rest of the steps and click **Enable.**
+5. Go the `index.html ` file of your React application
+6.  Paste our browser code snippet script tag right before the body tag ends (`</body>`).
+8. Restart your application. 
+
+Wait five minutes and then proceed to finding your data in New Relic. 
+
+## Find your data [#find-data]
+
+To get your MExN dashboard: 
+
+1. From [one.newrelic.com](https://one.newrelic.com), go to the [**Add data** page](https://one.newrelic.com/marketplace). 
+2. Click on **Dashboards**.
+3. In the search bar, type `MExN`. 
+4. The MExN dashboard should appear. Click on it to install it.
+
+Your MExN dashboard is considered a custom dashboard and can be found in the **Dashboards** UI. For docs on using and editing dashboards, see [our dashboard docs](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards).
+
+This integration reports data in the form of our [browser monitoring events](/docs/data-apis/understand-data/event-data/events-reported-browser-monitoring). If you installed the infrastructure agent, you'll also receive [infrastructure data](/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data/#infrastructure-events). 
```

Comment:
if they're also using the MongoDB integration, I assume that means they'd also be getting events reported by that integration, which you can find in that doc. From what I'm seeing in https://docswebsitedevelop-sdmexnintegration.gatsbyjs.io/docs/infrastructure/host-integrations/host-integrations-list/mongodb/mongodb-monitoring-integration-new/#find-and-use, it seems like maybe it's `Metric` data. maybe just link to that for more info. that doc is confusing to me way it's written.